demo/csi/aws/efs/example.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_kms_alias" "elasticfilesystem" {
+  name = "alias/aws/elasticfilesystem"
+}
+
+resource "aws_efs_file_system" "mytestefsvol" {
+  creation_token = "1eab7a71-2457-49b0-a56b-3519b21009a8"
+
+  encrypted = true
+  kms_key_id = data.aws_kms_alias.elasticfilesystem.target_key_arn
+
+  #lifecycle_policy {
+  #  transition_to_ia = AFTER_90_DAYS
+  #}
+
+  tags = {
+    Name = "my-test-efs-volume"
+  }
+}
+
+resource "aws_efs_mount_target" "mytestefsvol_mount" {
+  file_system_id = aws_efs_file_system.mytestefsvol.id
+  subnet_id      = aws_subnet.mysubnet.id
+  ip_address     = ...
+  security_groups = [
+    ...
+  ]
+}
+
+output "mytestefsvol" {
```

Comment:
Another option here that could make this a little more streamlined is to register the volume via Terraform directly, similar to how it's done in the DO example: https://github.com/hashicorp/nomad/blob/master/demo/csi/digitalocean/main.tf#L18-L28

Reply:
I'll provide examples of both (and make it clear it's either/or).